features/order/placing_an_order/adding_notes_for_customer_service.feature

```diff
@@ -0,0 +1,23 @@
+@placing_orders
+Feature: Adding a note to order
+    In order to provide some extra information to order
+    As a Customer
+    I want to be able to provide a note for customer service
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And the store has customer service account identified by "sylius@example.com"
```

Comment:
`customer service account` let's not introduce another term for administrator user :) Should be renamed to admin.